src/fs_report.rs

```diff
@@ -50,7 +50,7 @@ fn write_ingredient_thumbnails(ingredients: &[Ingredient], destination: &Path) -
         .iter()
         .filter_map(|i| {
             i.thumbnail().map(|(format, bytes)| {
-                let title = add_extension(i.title(), format);
+                let title = add_extension(i.instance_id(), format).replace(['/', ':'], "-");
```

Comment:
Hey, @timmurphydev, thanks and good catch! What are your thoughts on tracking the file names we've seen, and if we see a duplicate, append the `instance_id` to the `title`? We could also just append a number instead of the instance_id. I'd like to keep the title in the file name for UX purposes. @gpeacock, thoughts?

Reply:
@dyro, great point about the UX for command line users!

There were a couple of reasons I went with the instance_id to solve the missing thumbnail issue.  One was to reduce thumbnail/ingredient ambiguity for CLI users when there was a duplicate title.  The second was so server side users could still construct a path programmatically to specific thumbnails from manifest data.

What do you think about a filename based on the title (incremented number for duplicates and spaces removed) and then output json data that represents the directory structure and associates instance_ids with thumbnail paths?  Given that ingredient formats can be different from thumbnail formats (SVGs, PNGs w/o transparency), this also makes it easier to access specific thumbnails for server side users.

Either way, the feature is awesome!  Thanks for your work on it!

Reply:
The logic to generate the manifest store report with thumbnails is now integrated into c2pa-rs. As currently implemented it no longer puts ingredient files in folders. Manifest thumbnails are identified with the manifest label and the associated jpg or png extension. Ingredients are written using a file system safe modified version of the instance_id from the ingredient. So we currently use this suggested model and I think I can close this request.
The thumbnail file names are embedded into the manifest store json data in the thumbnail identifier field. If you are reading the thumbnails you should use the identifier values to match the thumbnail to the associated manifest or ingredient.

Reply:
closing based on the integration of the fix as described above.